### DISCUSSION TITLE
Is there any example on how to use the Depth Of Field effect?

### DISCUSSION THREAD

**Discussion by chelvisgit** [DiligentGraphics/DiligentEngine#297] [Q&A] [answered]
I added the DOF effect as the https://github.com/DiligentGraphics/DiligentFX/tree/master/PostProcess/DepthOfField shows but unfortunately I'm always ended up with a full blurred image. And yes, the depth and color buffer are sent correctly.

Any hint? Thanks

**Comment by MikhailGorobets** [comment, marked as answer]
@chelvisgit 
The current implementation of Depth of Field requires a physical сorrect camera. Please try to set the corresponding parameters as in this commit
https://github.com/DiligentGraphics/DiligentSamples/commit/de3d88a14abed83d34f2ee67295a291a275239e3

**Reply by chelvisgit** [reply]
Thank you!!
